src/openrct2/rct2/S6Importer.cpp

```diff
@@ -453,12 +454,25 @@ class S6Importer final : public IParkImporter
         if (String::Equals(_s6.scenario_filename, "Europe - European Cultural Festival.SC6"))
         {
             // This scenario breaks pathfinding. Create passages between the worlds. (List is grouped by neighbouring tiles.)
```

Comment:
With clang-format putting everything on one line, the list is no longer grouped.
This can be solved by using clang-format off, or by changing the comment.